subtests/docker_cli/systemd/systemd.py

```diff
@@ -0,0 +1,81 @@
+"""
+Summary
+-------
+
+Operational Summary
+---------------------
+    1. Copy Unit file
+    2. Build image using predefined Dockerfile & p4321-server.py
+    3. Run image
+    4. Confirm p4321-server script is running
+    5. Stop container
+    6. cleanup
+
+"""
+
+from dockertest import subtest
+from shutil import copy
+from os.path import join
+from autotest.client import utils
+import time
+from dockertest.dockercmd import DockerCmd
+
+
+class systemd(subtest.Subtest):
+
+    def initialize(self):
+        super(systemd, self).initialize()
+        sysd_unitf = join(self.bindir, self.config['sysd_unit_file'])
+        dockerfilesrc = join(self.bindir, 'Dockerfile')
+        script_src = join(self.bindir, 'p4321-server.py')
+        dockerbuild_dir = join(self.bindir, self.config['tmpblddir'])
+        self.config['dockerbuild_dir'] = dockerbuild_dir
+        self.config['unitf_dest'] = join(self.config['sysd_dest'],
+                                        self.config['sysd_unit_file'])
+        utils.run("mkdir %s" %dockerbuild_dir, ignore_status=True)
+        copy(sysd_unitf, self.config['sysd_dest'])
```

Comment:
Future maintainers will thank you if you make the destination be `unitf_dest` (i.e. the filename), not sysd_dest (directory name). And again, `self.sub_stuff[]` not config.